policyengine_us/variables/gov/states/nm/tax/income/credits/dependent_day_care_credit/nm_cdcc.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class nm_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico dependent child day care credit"
+    defined_for = StateCode.NM
+    unit = USD
+    definition_period = YEAR
+    reference = "https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/856ebf4b-3814-49dd-8631-ebe579d6a42b/Personal%20Income%20Tax.pdf"  # p63
+
+    def formula(tax_unit, period, parameters):
+        spm_unit = tax_unit.spm_unit
+        p = parameters(period).gov.states.nm.tax.income.credits.cdcc
+        # Maximum New Mexico CDCC amount
+        nm_cdcc_max = tax_unit("nm_cdcc_max_amount", period)
+        # Federal child and dependent care credit
+        fed_cdcc = tax_unit("cdcc", period)
+        # Filer has to be be gainfully employed to receive credit
+        employed = tax_unit("tax_unit_earned_income", period) > 0
+        # Filer can not receive tanf to be eligible
+        tanf = spm_unit("tanf", period)
```

Comment:
Only need `tax_unit.sum` for person-level variables. Since spm_unit is a parent of tax_unit, we can grab the actual value.
```suggestion
        receives_tanf = tax_unit.spm_unit("tanf", period) > 0
```